CMakeLists.txt

```diff
@@ -401,7 +401,7 @@ if(WITH_CORE)
   endif()
 
   set (WITH_QTWEBKIT TRUE CACHE BOOL "Enable QtWebkit support")
-  if (WITH_QTWEBKIT)
+  if (WITH_QTWEBKIT AND NOT BUILD_WITH_QT6)
```

Comment:
I think we should instead explicitly fail, can you add a condition on these two and `MESSAGE(FATAL_ERROR "WITH_QTWEBKIT is not supported for Qt6 builds")`.

Reply:
Maybe a WARNING + Forcing it to FALSE, it default to TRUE so It's not user/developer fault!

Reply:
That or changing the default for Qt6 are both fine for me :+1: